json_annotation/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 4.9.1-wip
```

Comment:
I'll do the follow-up here. We should bump this to `4.10` since we're adding a feature.